code/game/mecha/equipment/tools/mining_tools.dm

```diff
@@ -86,8 +86,11 @@
 		var/obj/item/organ/limb/affecting = H.get_organ("chest")
 		affecting.take_damage(drill_damage)
 		H.update_damage_overlays(0)
+	else if(target.stat == DEAD && target.butcher_results)
+		target.harvest(chassis) // Butcher the mob with our drill.
```

Comment:
The harvest proc is expecting a /mob/living as argument, not a mech, this may cause problems in the future... just saying.